### ISSUE TITLE
BASE_URL not working for admin 

### ISSUE THREAD

**Issue by amit-ch** [martialblog/docker-limesurvey#49]
I tried using the latest docker image and set base url to '/survey' and it works if I navigate to http://localhost/survey. However if I try admin url, its not following base url. Whenever I try to navigate to http://localhost/survey/admin It redirects with 301 to http://localhost/admin which returns 404. I have a requirement to serve limesurvey on a base path like https://server.com/base_path
Am I missing some config. Below is the docker-compose.yml I am using to try running locally.

```
version: "3.0"
services:
  limesurvey:
    image: martialblog/limesurvey
    restart: always
    volumes:
      - /tmp/upload/surveys:/var/www/html/upload/surveys
    labels:
      # Explicitly tell Traefik to expose this container
      - "traefik.enable=true"
      # The domain the service will respond to
      - "traefik.http.routers.limesurvey.rule=Host(`localhost`)"
      # Allow request only from the predefined entry point named "web"
      - "traefik.http.routers.limesurvey.entrypoints=web"
      - "traefik.http.routers.limesurvey.rule=PathPrefix(`/survey`)"
      - "traefik.http.routers.limesurvey.middlewares=strip-limesurvey@docker"
      - "traefik.http.middlewares.strip-limesurvey.stripprefix.prefixes=/survey"
    links:
      - lime-db
    depends_on:
      - lime-db
    expose: 
      - "80"
    environment:
      - "DB_HOST=lime-db"
      - "DB_PASSWORD=secret"
      - "ADMIN_PASSWORD=foobar"
      - "BASE_URL= /survey"
  lime-db:
    image: mysql:5.7
    environment:
      - "MYSQL_USER=limesurvey"
      - "MYSQL_DATABASE=limesurvey"
      - "MYSQL_PASSWORD=secret"
      - "MYSQL_ROOT_PASSWORD=secret"

  reverse-proxy:
    # The official v2 Traefik docker image
    image: traefik:v2.4
    # Enables the web UI and tells Traefik to listen to docker
    command:
      - "--log.level=DEBUG"
      - "--api.insecure=true"
      - "--providers.docker=true"
      - "--providers.docker.exposedbydefault=false"
      - "--entrypoints.web.address=:80"
    ports:
      # The HTTP port
      - "80:80"
      # The Web UI (enabled by --api.insecure=true)
      - "8080:8080"
    volumes:
      # So that Traefik can listen to the Docker events
      - /var/run/docker.sock:/var/run/docker.sock
```

**Comment by martialblog** [comment]
Hi, I'll have a look at it.

**Comment by martialblog** [comment]
The Base URL needs to be full-qualified. Worked for me. 

`"BASE_URL=http://localhost/survey"`

**Comment by amit-ch** [comment]
weird as same code with fully qualified base url not working for me.
I am able to navigate to http://localhost/survey fine but when I am trying to access **http://localhost/survey/admin** its getting 301 to **http://localhost/admin**
Can you confirm you are able to navigate fine http://localhost/survey/admin with above setup and base url as mentioned by you

**Comment by martialblog** [comment]
Hey, 

Ok so calling http://localhost/survey/index.php/admin/authentication/sa/login the first time works, however there is something fishy going on with the base_url. I'll look into it.

Cheers, Markus

**Comment by leonch123** [comment]
Hey,

I looked at how [crramirez](https://github.com/crramirez/limesurvey/blob/master/apache_default) handled the apache config in his package, since I've used it before with a traefik setup.

I added the `Alias /limesurvey /var/www/html` line, this fixed my asset loading error when navigating to /index.php/admin/authentication/sa/login manually. But replacing the whole file with ccramirez' one (adjusting for Alias path and :80 port) didn't fix the missing redirect from `/admin` to `/index.php/admin`. For reference, I configured my base_url to the full host including `/limesurvey` path and disabled my traefik middleware for stripping the prefix.

I just tested this inside the docker container using exec, adding directly to `/etc/apache2/sites-enabled/000-default.conf` and reloading with `service apache2 reload`. I'm not sure how to add this to the Dockerfile cleanly, unfortunately.
